cmd/neofs-node/config/node/config.go

```diff
@@ -0,0 +1,77 @@
+package nodeconfig
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+	"github.com/nspcc-dev/neofs-node/pkg/network"
+)
+
+const (
+	subsection = "node"
+
+	KeyDefault = ""
+
+	attributePrefix = "attribute"
+)
+
+var ErrEmptyAddress = errors.New("empty bootstrap address, see `node.address` section")
+
+// Key returns value of "key" config parameter
+// from "node" section.
+//
+// Returns KeyDefault if value is not set.
+func Key(c *config.Config) string {
+	v := config.StringSafe(c.Sub(subsection), "key")
+	if v != "" {
+		return v
+	}
+
+	return KeyDefault
+}
+
+// BootstrapAddress returns value of "address" config parameter
+// from "node" section and tries to convert it to network.Address.
+//
+// Panics if value is not set.
+func BootstrapAddress(c *config.Config) *network.Address {
+	v := config.StringSafe(c.Sub(subsection), "address")
+	if v == "" {
+		panic(ErrEmptyAddress)
+	}
+
+	addr, err := network.AddressFromString(v)
+	if err != nil {
+		panic(fmt.Errorf("could not convert bootstrap address to `network.Address`: %w", err))
```

Comment:
It would be nice to print original string there.
```suggestion
		panic(fmt.Errorf("could not convert bootstrap address %s to `network.Address`: %w", v, err))
```